.github/workflows/codeql.yml

```diff
@@ -48,11 +50,14 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v4
 
+

```

Comment:
newlines?

Reply:
we can fix it in mu_devops for the next file sync.